pupil_src/player/vis_light_points.py

```diff
@@ -46,15 +46,16 @@ def update(self,frame,events):
             except:
                 pass
 
-        out = cv2.distanceTransform(overlay,cv2.cv.CV_DIST_L2, 5)
+        out = cv2.distanceTransform(overlay,cv2.DIST_L2, 5)
 
         # fix for opencv binding inconsitency
         if type(out)==tuple:
             out = out[0]
 
         overlay =  1/(out/falloff+1)
 
-        img *= cv2.cvtColor(overlay,cv2.COLOR_GRAY2RGB)
+        img_float = img.astype(np.float32) * cv2.cvtColor(overlay,cv2.COLOR_GRAY2RGB)
+        img.data = img_float.round().astype(np.uint8).data
```

Comment:
This is a dirty hack, but I couldn't get it to work in a nicer way. Does anyone have an idea on how to implement this in a nicer way?